ports/psoc6/tests/vfs_fat.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
Wouldn´t these test belong to test/psoc6 ? or even there is some generic file system tests that we can run?
Any changed required to be added to the automated check?

Reply:
These tests were thought to be made examples for the user, but are not yet finalized. Then they could also be used for run_tests.py. 
So, shall I remove until a final version is available ?

Reply:
Mmm I would not remove them...  
I think the primary user, unless pointed to the repo would not look for examples here. But mpy developers or maintainers could take the tests as examples... 
At the same time, a primary user example does not necessarily needs to cover all the module functionality. And in a test, we might end having also some verification of not sunny side paths, forcing failures and so on. So some parts might not be useful in terms of explaining how to use the module to the user... But still, I would consider for now the tests as a big example for the complete (or as complete as possible) functionality of the module.

Reply:
Agreed, but now what next ?

Reply:
If they are ready to be run under ci/cd, I would add them to the test folder. If not we can just keep them where they are and do that in an upcoming ticket ?